charts/devnet/templates/relayers/go-relayer/statefulset.yaml

```diff
@@ -32,10 +32,7 @@ spec:
         app.kubernetes.io/name: {{ $relayer.type }}-{{ $relayer.name }}
         app.kubernetes.io/version: {{ $.Chart.AppVersion }}
     spec:
-      imagePullSecrets:
-      {{- range $imagePullSecrets }}
-        - name: {{ .name }}
-      {{- end }}
+      {{- include "imagePullSecrets" $relayer | indent 6 }}
```

Comment:
Since this is added to go-relayer, it might just make sense to add it to other relayers, explorers and others too.
What do you think?